src/app/plugins/connectfour.plugin.ts

```diff
@@ -171,12 +133,12 @@ class ConnectFourGame {
 
   public getWinner(): User {
     // -1 is playerA
-    return this._winner === -1 ? this._playerA : this._playerB;
+    return this._winner === -1 ? this.playerA : this.playerB;
```

Comment:
This assumes that someone won
There can be a tie, and should prob return null if so

Reply:
Agreed